orbs/pulumi.yml

```diff
@@ -50,6 +50,42 @@ commands:
             export PULUMI_ACCESS_TOKEN="<< parameters.access-token >>"
             pulumi login --cloud-url << parameters.cloud-url >>
 
+  # stack init command
+  stack_init:
+    parameters:
+      stack:
+        type: string
+        description: "Name of the stack to initialize."
+      secrets_provider:
+        type: string
+        description: "The type of the provider that should be used to encrypt and decrypt secrets (possible choices: default, passphrase, awskms, azurekeyvault, gcpkms, hashivault) (default "default")."
+        default: "default"
+      working_directory:
+        type: string
+        description:
+            "The relative working directory to use, i.e. where your Pulumi.yaml is located."
+        default: "."
+    steps:
+      - run:
+          name: "pulumi stack init --stack << parameters.stack >> --secrets-provider << parameters.secrets_provider >>"
+          command: pulumi stack init --stack << parameters.stack >> --secrets-provider << parameters.secrets_provider >> --cwd << parameters.working_directory >>
+
+  # stack rm command
+  stack_rm:
+    parameters:
+      stack:
+        type: string
+        description: "Name of the stack to remove."
+      working_directory:
+        type: string
+        description:
+            "The relative working directory to use, i.e. where your Pulumi.yaml is located."
+        default: "."
+    steps:
+      - run:
+          name: "pulumi stack rm << parameters.stack >>"
+          command: pulumi stack rm << parameters.stack >> --force --yes --cwd << parameters.working_directory >>
```

Comment:
Since this is inside a CI, I've add `--force` and `--yes`. Was this appropriate??

Reply:
It's a reasonable suggestion, but we probably don't want to add those by default.

We should definitely make `--force` a parameter to `stack_rm`, since if the stack still has cloud resources, then passing force will orphan them. (Meaning you'd have random resources in your AWS, GCP account you'd pay for with no knowledge of how they got there.)

So making `--force` optional means that if that's the behavior you want, you need to explicitly opt-into it in your CircleCI configuration.